packages/jest-matchers/src/spy-matchers.js

```diff
@@ -44,6 +45,24 @@ const spyMatchers: MatchersObject = {
 
     return {message, pass};
   },
+
+  toHaveBeenCalledWith(actual: any, expected: any) {
+    ensureSpy(actual, 'toHaveBeenCalledWith');
+
+    const args = Array.from(arguments).pop().args;
```

Comment:
this fill be cleaned up after https://github.com/facebook/jest/pull/1298/files merged